src/Extracting/Strategies/ResponseParameters/CollectTransformerParamsHelper.php

```diff
@@ -44,10 +44,16 @@ protected function fromTransformMethod(ReflectionClass $transformerReflection)
      */
     protected function collectIncludes(ReflectionClass $transformerReflection): array
     {
-        $includes = $transformerReflection->getProperty('defaultIncludes');
-        $includes->setAccessible(true);
-        $includes = $includes->getValue($transformerReflection->newInstanceWithoutConstructor());
+        $defaultIncludes = $transformerReflection->getProperty('defaultIncludes');
+        $defaultIncludes->setAccessible(true);
+        $defaultIncludes = $defaultIncludes->getValue($transformerReflection->newInstanceWithoutConstructor());
 
+        $availableIncludes = $transformerReflection->getProperty('availableIncludes');
+        $availableIncludes->setAccessible(true);
+        $availableIncludes = $availableIncludes->getValue($transformerReflection->newInstanceWithoutConstructor());
+
+        $includes = $defaultIncludes + $availableIncludes;
```

Comment:
```suggestion
        $includes = array_merge($defaultIncludes, $availableIncludes);
```

`+` only can be trusted if you the arrays always have string keys, otherwise you get annoying overwriting behaviour. Pretty sure includes can be both non-associative and associative so `array_merge` is better in this case.